src/src/AdminBundle/Entity/Event.php

```diff
@@ -0,0 +1,205 @@
+<?php
+
+namespace AdminBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use AdminBundle\Entity\EventType;
+use Gedmo\Mapping\Annotation as Gedmo;
+
+/**
+ * Event
+ *
+ * @ORM\Table(name="event")
+ * @ORM\Entity(repositoryClass="AdminBundle\Repository\EventRepository")
+ */
+class Event extends BaseEntity
+{
+    /**
+     * @var int
+     *
+     * @ORM\Column(name="id", type="integer")
+     * @ORM\Id
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    private $id;
+
+
+    /**
+     * Get id
+     *
+     * @return int
+     */
+    public function getId()
+    {
+        return $this->id;
+    }
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(name="address", type="text")
+     */
+    private $address;
+
+    /**
+     * @var \DateTime
+     *
+     * @Gedmo\Timestampable(on="update")
```

Comment:
hmmm.... Niesom si isty, ale mam pocit, ze `@Gedmo\Timestampable(on="update")` toto pri kazdom update tento cas zmeni. To nieje to co chceme, mi tu chceme mat cas kedy sa bude udalost konat a teda sa to ma nastavit rucne. Tento riadok teda treba zmazat.